main.go

```diff
@@ -120,6 +122,8 @@ func main() {
 		"The TTL of an index in the cache. Valid time units are ns, us (or µs), ms, s, m, h.")
 	flag.StringVar(&helmCachePurgeInterval, "helm-cache-purge-interval", "1m",
 		"The interval at which the cache is purged. Valid time units are ns, us (or µs), ms, s, m, h.")
+	flag.StringSliceVar(&kexAlgos, "kex-algos", []string{},
```

Comment:
```suggestion
	flag.StringSliceVar(&kexAlgos, "ssh-kex-algos", []string{},
```

this allows for prefix grouping if the list of arguments grows (`git-` might be an alternative).